Off-Top-App/off-top-flutter
# OT-RedirectReport | Hid the reports graph until clicked upon a row on the reports table

**What I did**: 
I hid the reports graph until you click on a row on the reports table and the graph will pop up displaying the date associated with the row you clicked on as a way to differentiate the rows. There is also a back button so you can go back to the reports table. 

**How to test**: 
**Have EVERYTHING set up to run the OFF-TOP app**.

1. Have OFF-TOP running.
2. Click on the reports tab and you will see the reports table.
Click on any row you want to see.
![Screenshot (48)](https://user-images.githubusercontent.com/55634110/89604783-d5752a00-d820-11ea-8668-621a1c1376ee.png)
4. You will see the reports graph pop up and the title is the same date as the row you clicked on.
To return to the reports table, click on the back arrow button on the top left corner.
![Screenshot (49)](https://user-images.githubusercontent.com/55634110/89604801-dc03a180-d820-11ea-9314-e4fddb89de2c.png)
